contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java

```diff
@@ -120,9 +120,9 @@ public void run() throws Exception {
         }
 
         AdminClientConfig adminConfig = new AdminClientConfig().setMaxConnectionsPerNode(cluster.getNumberOfNodes())
-                                                               .setAdminConnectionTimeoutSec(15)
+                                                               .setAdminConnectionTimeoutSec(60)
```

Comment:
The default connection timeout is 60s, we don't need to specify it if we want to use this value:
https://github.com/squarY/voldemort/blob/master/src/java/voldemort/client/protocol/admin/AdminClientConfig.java#L17